about.html

```diff
@@ -67,10 +69,21 @@ <h1>
            <h3> See the past Vortech Global summits</h3>
         <p>take a look at the last two Vortech Engineering Global summits which took place in Kiev Ukraine and in Georgia</p>
         <div class="about-row">
-            <div class="about-row-col-2"><img src="./images/conference_1.jpg" alt="conference"></div><div class="about-row-col-2"><img src="./images/conference_2.jpg" alt="conference two"></div>
+            <div class="about-row-col-2">
+                <div class="layer">
+                    <h6>2016 <br>VE Annual Global summit, Ukraine</h6>
+                </div>
+                <img src="./images/conference_1.jpg" alt="conference">
+                 </div>
+            <div class="about-row-col-2">
+                <div class="layer">
+                    <h6>2011 <br>VE Annual Global summit, Georgia</h6>
+                </div>
+                <img src="./images/conference_2.jpg" alt="conference two">
+            </div>
         </div>
        </section>
-       <footer class="black-bg"><img src="./images/FOOTER-IMG.jpeg" alt="footer logo"><div><p class="white">Vortech Engineering   USA. All rights Reserved</p><p class="white">Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure voluptatum ea quam <br> laboriosam architecto eaque cumque non fuga voluptas praesentium.</p></div></footer>
+       <footer><img src="./images/footer_image.jpeg" alt="footer logo"><div><p>Vortech Engineering   USA. All rights Reserved</p><p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure voluptatum ea quam <br> laboriosam architecto eaque cumque non fuga voluptas praesentium.</p></div></footer>
         
```

Comment:
- [ ] As requested by the previous reviewer here 👇🏻 

> 
> - [ ] For the mobile version footer kindly ensure it contains a logo aligned to the left and text aligned to the right.
  > 
  > | Your Design | The Template |
  > |---------------|----------------|
  > |![Screenshot 2022-05-20 012040](https://user-images.githubusercontent.com/77142475/169419913-fa930965-41d6-4e23-9185-29b8cc40c239.png)|![Screenshot 2022-05-20 012130](https://user-images.githubusercontent.com/77142475/169419932-32f2e8e1-b57c-4127-8dcd-93d0f3adea41.png)|
> 
> - [ ] In the desktop version kindly ensure that the About Page footer has a black background 👍🏻 
> - [ ] In the desktop version About page footer, kindly align the text and the logo. You could make reference the screenshots below.

 > | Your Design | The Template |
  > |---------------|----------------|
  > |![footerwhit](https://user-images.githubusercontent.com/79058364/169441947-63c612ec-09c5-4938-bd24-6b7b724ac7d5.png)|![Screenshot 2022-05-20 013008](https://user-images.githubusercontent.com/77142475/169420599-56b1e4c0-40f4-48b1-a08a-7ad412f3cd13.png)|
> 
 
  
_Originally posted by @chaw-bot in https://github.com/Emmykage/Capstone-Project/pull/1#discussion_r877614718_

Reply:
the last reviewer said i should make it white and now i have made it white now you want me to make it black again?